saleor/plugins/webhook/tasks.py

```diff
@@ -139,3 +167,32 @@ def send_webhook_request(webhook_id, target_url, secret, event_type, data):
         target_url,
         event_type,
     )
+
+
+def send_webhook_request_sync(target_url, secret, event_type, data: str):
+    parts = urlparse(target_url)
+    domain = Site.objects.get_current().domain
+    message = data.encode("utf-8")
+    signature = signature_for_payload(message, secret)
+
+    response_data = None
+    if parts.scheme.lower() in [WebhookSchemes.HTTP, WebhookSchemes.HTTPS]:
+        logger.debug(
+            "[Webhook] Sending payload to %r for event %r.", target_url, event_type
+        )
+        try:
+            response = send_webhook_using_http(
+                target_url, message, domain, signature, event_type
+            )
+            response_data = response.json()
+        except RequestException as e:
+            logger.debug("[Webhook] Failed request to %r: %r.", target_url, e)
+        except JSONDecodeError as e:
+            logger.debug(
+                "[Webhook] Failed parsing JSON response from %r: %r.", target_url, e
+            )
+        else:
+            logger.warning("[Webhook] Success response from %r.", target_url)
```

Comment:
Are you sure that we are going to WARNING all success response? 😄

Reply:
:policeman: